src/types/generated/models.ts

```diff
@@ -4776,7 +4776,7 @@ export type IndicatorType = {
     created: string
     createdBy: User
     displayName: string
-    factor: number
+    factor: string
```

Comment:
This is actually a number from the API. So please don't change this. Ref: https://debug.dhis2.org/dev/api/schemas/indicatorType (search for `factor`). Or eg. https://debug.dhis2.org/dev/api/indicatorTypes?fields=displayName,id,factor .
These type reflect the API, and we should instead convert the values in the field.